src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -235,14 +235,15 @@ public class Thread implements Runnable {
     private volatile ClassLoader contextClassLoader;
 
     // Additional fields for platform threads.
-    // All fields, except task, are accessed directly by the VM.
+    // All fields, except task and terminatingThreadLocals, are accessed directly by the VM.
     private static class FieldHolder {
         final ThreadGroup group;
         final Runnable task;
         final long stackSize;
         volatile int priority;
         volatile boolean daemon;
         volatile int threadStatus;
+        ThreadLocal.ThreadLocalMap terminatingThreadLocals;
```

Comment:
I think it's worth documenting the memory safety aspects of this field.

Reply:
Comment added to make it clear that it is maintained by the TL class.